examples/hitl/basic_viewer/config/basic_viewer.yaml

```diff
@@ -2,18 +2,26 @@
 
 defaults:
   - social_rearrange: pop_play
+  - hitl_defaults
   - _self_
 
 habitat:
   environment:
     iterator_options:
-      cycle: False
+      # for manual inspection of episodes by a user, it makes sense to disable shuffle
+      shuffle: False
 
 habitat_baselines:
-  evaluate: True
-  num_environments: 1
+  # todo: document these choices
```

Comment:
We have a lot of documentation todos for the HITL framework which we're tracking [here](https://app.asana.com/0/1205829436198608/1206145143211296).